Standups-StackBased/Standups/App.swift

```diff
@@ -0,0 +1,176 @@
+import Combine
+import Dependencies
+import SwiftUI
+
+@MainActor
+class AppModel: ObservableObject {
+  @Published var path: [Destination] {
+    didSet { self.bind() }
+  }
+  @Published var standupsList: StandupsListModel {
+    didSet { self.bind() }
+  }
+
+  @Dependency(\.continuousClock) var clock
+  @Dependency(\.date.now) var now
+  @Dependency(\.uuid) var uuid
+
+  private var detailCancellable: AnyCancellable?
+
+  enum Destination: Hashable {
+    case detail(StandupDetailModel)
+    case meeting(Meeting, standup: Standup)
+    case record(RecordMeetingModel)
+  }
+
+  init(
+    path: [Destination] = [],
+    standupsList: StandupsListModel
+  ) {
+    self.path = path
+    self.standupsList = standupsList
+    self.bind()
+  }
+
+  private func bind() {
+    self.standupsList.onStandupTapped = { [weak self] standup in
+      guard let self else { return }
+
+      self.path.append(
+        .detail(
+          StandupDetailModel(standup: standup)
+        )
+      )
+    }
+
+    for destination in self.path {
+      switch destination {
+      case let .detail(detailModel):
+        detailModel.onMeetingStarted = { [weak self] standup in
+          guard let self else { return }
+
+          self.path.append(
+            .record(
+              RecordMeetingModel(standup: standup)
+            )
+          )
+        }
+        detailModel.onConfirmDeletion = { [weak detailModel, weak self] in
+          guard let self, let detailModel else { return }
+          self.standupsList.standups.remove(id: detailModel.standup.id)
+          _ = self.path.popLast()
+        }
+        detailModel.onMeetingTapped = { [weak detailModel, weak self] meeting in
+          guard let self, let detailModel else { return }
+
+          self.path.append(.meeting(meeting, standup: detailModel.standup))
+        }
+
+        self.detailCancellable = detailModel.$standup
+          .sink { [weak self] standup in
+          self?.standupsList.standups[id: standup.id] = standup
+        }
+
+      case .meeting:
+        break
+
+      case let .record(recordModel):
+        recordModel.onDiscardMeeting = { [weak self] in
+          guard let self else { return }
+          _ = self.path.popLast()
```

Comment:
Same here?

```suggestion
          self?.path.removeLast()
```

Or:

```suggestion
          _ = self?.path.popLast()
```